changelog.d/3-bug-fixes/WPB-5064

```diff
@@ -0,0 +1 @@
+Updated from deprecated docker image namshi/smtp to ixdotai/smtp
```

Comment:
```suggestion
Updated `demo-smtp` Helm chart from deprecated docker image namshi/smtp to ixdotai/smtp
```

Most customers are using their own SMTP server. So, it might be worth noting what this change is about (the "demo-smtp" server.)